test/extended/router/http2.go

```diff
@@ -143,23 +217,32 @@ var _ = g.Describe("[sig-network-edge][Conformance][Area:Networking][Feature:Rou
 				useHTTP2Transport: false,
 			}}
 
-			for i, tc := range testCases {
-				testConfig := fmt.Sprintf("%+v", tc)
-				e2e.Logf("[test #%d/%d]: config: %s", i+1, len(testCases), testConfig)
+			g.By("Getting LB service")
+			shardService, err := getRouterService(oc, 10*time.Minute, "router-"+oc.Namespace())
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(shardService).NotTo(o.BeNil())
+			o.Expect(shardService.Status.LoadBalancer.Ingress).To(o.Not(o.BeEmpty()))
 
-				// check readiness probe is accessible
-				urlTester := url.NewTester(oc.AdminKubeClient(), oc.KubeFramework().Namespace.Name).WithErrorPassthrough(true)
-				defer urlTester.Close()
-				hostname := getHostnameForRoute(oc, tc.route)
-				urlTester.Within(30*time.Second, url.Expect("GET", "https://"+hostname+"/healthz").Through(hostname).SkipTLSVerification().HasStatusCode(200))
+			g.By("Waiting for new DNS zone to register")
+			if len(shardService.Status.LoadBalancer.Ingress[0].IP) > 0 {
+				addrs, err := resolveHostAsAddress(oc, time.Minute, 15*time.Minute, testCases[0].route+"."+shardFQDN, shardService.Status.LoadBalancer.Ingress[0].IP)
+				o.Expect(err).NotTo(o.HaveOccurred())
+				o.Expect(addrs).NotTo(o.BeEmpty())
+			} else {
+				addrs, err := resolveHost(oc, time.Minute, 15*time.Minute, shardService.Status.LoadBalancer.Ingress[0].Hostname)
```

Comment:
Same Q as earlier..... does attempting to resolve the LB's hostname tell us anything about DNS for the route?

Reply:
My assumption and, in practice, assertion is that as soon as <anything>.new-domain resolves so will route name.

Reply:
As we discussed offline will also validate the DNS of a route for the case where we have a hostname after the LB name resolves.